README.md

```diff
@@ -90,6 +90,7 @@ This is the running list of what in tech has been affected by COVID-19. Pull req
 | Roblox | Required | Restricted | Restricted | Restricted | 2020-03-06 |
 | Salesforce[[1]](https://www.salesforce.com/blog/2020/03/safety-and-wellbeing-those-around-you.html) | Encouraged | Restricted | Restricted | Restricted | 2020-03-03 |
 | SAP | Encouraged | Restricted | Restricted | Restricted | 2020-03-05 |
+| Shopify[[1]](https://twitter.com/Shopify/status/1237741373417193476) | Encouraged | ? | ? | ? | 2020-03-11 |
```

Comment:
```suggestion
| Shopify[[1]](https://twitter.com/Shopify/status/1237741373417193476) | Required | Restricted | Restricted | Restricted | 2020-03-11 |
```
Coming to the offices is so highly discouraged, that it might as well be marked as required. The travel and event restrictions were already in place since last week. Example: https://twitter.com/Shopify/status/1233406636993187840